blockchain/blockchain-wallet-v4-frontend
# feat(cta-interest): implemented proper events

## Description (optional)
This PR brings on proper matomo events

## Testing Steps (optional)
Use gold tier 2 account, go over SB, once when there is a interest banner you can confirm in net tab that event has been dispatched
